pandas/core/strings.py

```diff
@@ -558,7 +558,7 @@ def str_replace(arr, pat, repl, n=-1, case=None, flags=0, regex=True):
 
     Returns
     -------
-    replaced : Series/Index of objects
+    Series/Index of objects
```

Comment:
I think we usually document this as `Series or Index of object` (note I also change objects to object).

Also, If you can add a description in the next line, indented with four spaces, explaining what this method returns, that would be great.